lib/engine/game/g_1860.rb

```diff
@@ -43,11 +52,79 @@ def init_stock_market
         StockMarket.new(self.class::MARKET, [], zigzag: true)
       end
 
+      def init_corporations(stock_market)
+        min_price = stock_market.par_prices.map(&:price).min
+
+        self.class::CORPORATIONS.map do |corporation|
+          Engine::G1860::Corporation.new(
+            min_price: min_price,
+            capitalization: self.class::CAPITALIZATION,
+            **corporation.merge(corporation_opts),
+          )
+        end
+      end
+
+      def new_auction_round
+        Round::Auction.new(self, [
+          # Step::CompanyPendingPar,
+          Step::G1860::BuyCert,
+        ])
+      end
+
+      def init_round_reorder_players
+        players_by_cash = @players.sort_by(&:cash).reverse
+
+        if players_by_cash[0].cash > players_by_cash[1].cash
+          player = players_by_cash[0]
+          reason = 'most cash'
+        else
+          # tie-breaker: lowest total face value in private companies
+          player = @players.select { |p| p.companies.any? }.min_by { |p| p.companies.sum(&:value) }
+          reason = 'least value of private companies'
+        end
+
+        @players.rotate!(@players.index(player))
+        @log << "#{player.name} has priority deal (#{reason})"
+      end
+
+      def next_round!
```

Comment:
what's the summary of changes here?

Reply:
I needed a new method of calculating the priority deal between the auction round and the first stock round.

Reply:
instead of doing all of that could you just override a different method like round_finished?

Reply:
I could have put it at the end of the auction round in the step logic, but that seemed out of place.

Reply:
Crap! I didn't realize there was an "init_round_finished" method.

Reply:
I'll just need to make sure the standard "reorder_players" doesn't re-reorder, or override that too.